rct229/rules/section22/section22rule16.py

```diff
@@ -0,0 +1,108 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all, find_exactly_one_with_field_value
+from rct229.utils.pint_utils import CalcQ
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
```

Comment:
The `get_baseline_system_types` function is merged. Please update the `APPLICABLE_SYS_TYPES` list and `is_applicable` function in this rule definition

Reply:
Fixed.